server/backend/courses/tests/test_models.py

```
@@ -0,0 +1,108 @@
+            description="Unique Desc",
+            created_by=instructor,
+            duration=5,
+            difficulty="advanced",
+            subject="Chemistry"
+        )
+        Enrollment.objects.create(course=course, student=student)
+
+        with pytest.raises(Exception):
+            Enrollment.objects.create(course=course, student=student)
+
+    def test_rating_only_enrolled_students(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_rating_only_enrolled_students`

Add return type annotation: `None`

(ANN201)